.pipelines/UpdateVersions.ps1

```diff
@@ -0,0 +1,126 @@
+Param(
```

Comment:
Do you mind to make this name more specific for updating WinAppSDK versions?
In the future, we may need other ps1 for update other version as well.